en-US/intro/faqs.md

```diff
@@ -32,7 +32,7 @@ server {
 ##### How do I set up a sub-path with NGINX?
 
 In case you need to use a sub-path for your Gogs instance, you can change your NGINX configuration to the following
-(note to the both suffixes `/` after **location** and **proxy_pass**):
+(note the `/` suffixes after **location** and **proxy_pass**):
```

Comment:
It is **extremely important** to point out word "both", so this word can not be deleted, please update, thanks!

Reply:
Thanks @Unknwon, I understand. I will update it soon.